AgentDiscoveries-Frontend/app/src/components/admin/location-form.jsx

```diff
@@ -72,7 +90,23 @@ export default class LocationForm extends React.Component {
     }
 
     onSiteChange(event) {
-        this.setState({ siteName: event.target.value });
+        if(event.target.value.length < 200){
+            this.setState({ siteName: event.target.value });
+        }
+    }
+
+    onLongitudeChange(event) {
+        if(event.target.value < 100 && event.target.value > -100){
```

Comment:
Should this be 100 & -100? [Google says](https://www.google.com/search?q=longitude+range&rlz=1C1GCEA_enGB762GB762&oq=longitude+range&aqs=chrome..69i57j0l5.1476j0j4&sourceid=chrome&ie=UTF-8) it should be -180 to 180, and latitude -90 to 90